packages/dds/ordered-collection/src/test/consensusOrderedCollection.spec.ts

```diff
@@ -61,30 +61,31 @@ function createCollectionForReconnection(
 }
 
 describe("ConsensusOrderedCollection", () => {
+	/* eslint-disable @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-assignment */
```

Comment:
Did we try to use `unknown[]` here? It's not public API so it should be a safe change to make.

Reply:
We could also make this function generic, assuming tests are doing anything other than equality comparisons with the input/output entries.